examples/abitest/tests/src/tests.rs

```diff
@@ -49,7 +49,7 @@ fn build_wasm() -> std::io::Result<Vec<(String, Vec<u8>)>> {
 fn test_abi() {
     simple_logger::init().unwrap();
 
-    let configuration = oak_tests::test_configuration(
+    let configuration = oak_runtime::config::application_configuration(
```

Comment:
If the intention is to use this for configuration of non-tests too, perhaps re-export this from `oak_runtime` top level? `oak_runtime::application_configuration`